src/draw/lv_draw.c

```diff
@@ -219,8 +219,8 @@ bool lv_draw_dispatch_layer(lv_display_t * disp, lv_layer_t * layer)
                     int32_t h = lv_area_get_height(&layer_drawn->buf_area);
                     uint32_t layer_size_byte = h * layer_drawn->draw_buf->header.stride;
 
-                    _draw_info.used_memory_for_layers_kb -= get_layer_size_kb(layer_size_byte);
-                    LV_LOG_INFO("Layer memory used: %" LV_PRIu32 " kB\n", _draw_info.used_memory_for_layers_kb);
+                    _draw_info.used_memory_for_layers -= layer_size_byte;
```

Comment:
Is guaranteed no underflow here?

Reply:
In theory underflow should happen, but doing something like this would make sense:
```c
if(_draw_info.used_memory_for_layers >= layer_size_byte) {
   _draw_info.used_memory_for_layers -= layer_size_byte;
} else {
   _draw_info.used_memory_for_layers = 0;
   LV_LOG_WARN("More layers are freed than allocated");
}
```

Reply:
Yeah I mean, should underflow case needs handling, something like you posted above :)

Reply:
Sorry, I meant "should **not** happen"

Reply:
Done